gatsby/src/components/situation-detail/situation-detail.tsx

```diff
@@ -111,18 +108,27 @@ const SituationDetail: React.FC<IProps> = ({ situation }) => {
             ''
           )}
 
-          <p className="mt-2 text-muted font-italic">
+          {/* <p className="mt-2 text-muted font-italic">
             Naposledy bylo toto téma aktualizováno {situation.changed}
-          </p>
+          </p> */}
```

Comment:
Zabudnuty comment?

Reply:
@Haaxor1689 aktuálně se má skrýt, mělo by se brzy přidat, tak jsem to nechtěl mazat